kotlin-antd/src/main/kotlin/antd/menu/SubMenu.kt

```diff
@@ -11,12 +11,15 @@ external interface SubMenuProps : RProps {
     var rootPrefixCls: String?
     var className: String?
     var disabled: Boolean?
+    var level: Number?
     var title: Any? /* String | ReactElement */
+    var icon: Any? /* ReactElement */
```

Comment:
same here.

Reply:
Same as Menu.kt, it's fine to do `var icon: ReactElement?`